parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -155,22 +155,26 @@ installStandaloneContainerd() {
     # azure-built runtimes have a "+azure" suffix in their version strings (i.e 1.4.1+azure). remove that here.
     CURRENT_VERSION=$(containerd -version | cut -d " " -f 3 | sed 's|v||' | cut -d "+" -f 1)
     # v1.4.1 is our lowest supported version of containerd
-    local CONTAINERD_VERSION="1.4.4"
+    local CONTAINERD_VERSION="1.5.0-beta.git31a0f92df"
     if semverCompare ${CURRENT_VERSION:-"0.0.0"} ${CONTAINERD_VERSION}; then
         echo "currently installed containerd version ${CURRENT_VERSION} is greater than (or equal to) target base version ${CONTAINERD_VERSION}. skipping installStandaloneContainerd."
     else
         echo "installing containerd version ${CONTAINERD_VERSION}"
         removeMoby
         removeContainerd
         updateAptWithMicrosoftPkg
-        apt_get_install 20 30 120 moby-containerd=${CONTAINERD_VERSION}* --allow-downgrades || exit $ERR_CONTAINERD_INSTALL_TIMEOUT
+        # TODO: first try downloading from microsoft apt repo then fallback to our storage ep
+        downloadContainerd ${CONTAINERD_VERSION}
+        wait_for_apt_locks
+        retrycmd_if_failure 10 5 600 apt-get -y -f install ${CONTAINERD_DEB_FILE} || exit $ERR_CONTAINERD_INSTALL_TIMEOUT
```

Comment:
we noticed issue recently that wait for apt lock will fail and needs retries. Our raw ubuntu e2e test case will fail in that case. 
If we often need to install from file, we might need a apt_get_install_from_file method where it retries wait_for_apt_locks like the current apt_get_install. With that we can also switch between the two method, or do 

apt_get_install 20 30 120 moby-containerd=${CONTAINERD_VERSION}* --allow-downgrades || (downloadContainerd ${CONTAINERD_VERSION} && (apt_get_install_from_file 20 30 120 ${CONTAINERD_DEB_FILE})  && break) || exit $ERR_CONTAINERD_INSTALL_TIMEOUT